api/openapi/model-registry.yaml

```diff
@@ -1736,6 +1860,166 @@ components:
+              description: Longer description of the model.
+            logo:
+              type: string
+              format: uri
+              description: |-
+                URL to the model's logo. A [data
+                URL](https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/data)
+                is recommended.
+            readme:
+              type: string
+              description: Model documentation in Markdown.
+            language:
+              type: array
+              description: List of supported languages (https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes).
+              items:
+                type: string
+              example:
+                - en
+                - es
+                - cz
+            license:
+              type: string
+              description: Short name of the model's license.
+              example: apache-2.0
+            licenseLink:
+              type: string
+              format: uri
+              description: URL to the license text.
+            maturity:
+              type: string
+              description: Maturity level of the model.
+              example: Generally Available
+            libraryName:
+              type: string
+              example: transformers
+            baseModel:
+              type: array
+              description: Reference to the base model (if any).
+              items:
+                type: object
+                properties:
+                  catalog:
+                    type: string
+                    description: |-
+                      Name of the catalog for an external base model. Omit for
+                      models in the same catalog.
+                    example: huggingface.io
+                  repository:
+                    type: string
+                    description: |-
+                      Name of the repository in an external catalog where the base
+                      model exists. Omit for models in the same catalog.
+                    example: ibm-granite
+                  name:
+                    type: string
+                    example: granite-3.1-8b-base
+            labels:
+              type: array
+              description: List of labels for categorization.
+              example:
+                - language
+              items:
+                type: string
+            tasks:
+              type: array
+              description: List of tasks the model is designed for.
+              items:
+                type: string
+              example:
+                - text-generation
+            artifacts:
+              type: array
+              description: |-
+                If a model has multiple versions, each version should have a
+                separate artifact.
+              items:
+                $ref: "#/components/schemas/CatalogModelArtifact"
+        - $ref: "#/components/schemas/BaseResourceDates"
+    CatalogModelArtifact:
```

Comment:
How does it differ with existing `ModelArtifact`?
If it is a similar concept I would apply the same comment as before: define a common type or extend one type to implement the other

Reply:
I would expect a `CatalolgModelArtifact` to have `name` and `description` at least.
At the same time `artifactType`, `modelFormatName`, `storageKey`, `storagePath`, `modelFormatVersion` can become common and maybe replace `protocol`.